pkgs/top-level/python-packages.nix

```diff
@@ -5465,6 +5465,8 @@ in {
 
   parquet = callPackage ../development/python-modules/parquet { };
 
+  parquet-tools = callPackage ../development/python-modules/parquet-tools { };
```

Comment:
Sorry for not realizing this earlier, but this piece of software is unlikely to be useful as a library. So it should be using `buildPythonApplication`, the `pythonPackages.parquet-tools` attribute should be removed and the nix file moved out of pkgs/development/python-modules.

Reply:
Thanks! Fixing this now.